types/modules/admob/Banner.d.ts

```diff
@@ -0,0 +1,4 @@
+/// <reference types="react" />
+import * as React from 'react';
+declare const Banner: React.SFC;
```

Comment:
Not sure what happened here, but `Banner` should be a `React.SFC<AdMobComponentProps>` (or just a regular `React.Component` or `React.PureComponent`)